lockfile/src/csharp.rs

```diff
@@ -95,11 +94,14 @@ pub struct CSProj;
 
 #[derive(Debug, Deserialize, PartialEq, Eq)]
 pub struct PackageReference {
-    #[serde(alias = "Include", default)]
+    #[serde(alias = "@Include", default)]
     pub name: String,
 
-    #[serde(alias = "Version", default)]
+    #[serde(alias = "@Version", default)]
     pub version: String,
+
+    #[serde(alias = "@version", default)]
+    pub version_lower: String,
```

Comment:
What is this? It sounds unrelated to the patch itself and sounds suspiciously like a manifest (opposed to lockfile) thing.

Reply:
Removed, didn't realize you could use multiple alias', but the test file has two version attributes (Version and version).